Xamarin.UITest.POPSample/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="NUnit" version="2.6.3" targetFramework="net45" />
```

Comment:
looks kinda weird though if the git history shows us downgrading from 2.6.4 to 2.6.3. And might as well use the latest we can.